layouts/default.vue

```diff
@@ -86,7 +86,7 @@
 			<Popover ref="notificationsPanel">
 				<Accordion v-if="reverseNotifications.length" class="box-border w-96 max-w-[calc(100vw-16px)]" expand-icon="pi pi-chevron-right">
 					<AccordionPanel v-for="notification in reverseNotifications" :key="notification.id" :value="notification.id" @click="markNotificationAsRead(notification.id)">
-						<AccordionHeader :class="{ '!font-normal': notification.status !== 'inbox' }">{{ notification.subject }}</AccordionHeader>
+						<AccordionHeader class="text-left" :class="{ '!font-normal': notification.status !== 'inbox' }">{{ notification.subject }}</AccordionHeader>
```

Comment:
What exactly is this supposed to change?

Reply:
Without the change, long enough subject is centered:

<img width="562" alt="image" src="https://github.com/user-attachments/assets/1287fd8e-ce34-44b8-995f-eceaebf47f08" />